src/arrays/index.md

```diff
@@ -84,15 +84,14 @@ note: library: util
 ```
 
 As well as compiling `averages.rs` into a static library, `rustc` has also
-emitted some helpful notes which we'll need when linking the library into one
-executable.
+emitted some helpful notes needed for linking the library into one executable.
```

Comment:
Yeah, I agree. In that context "an executable" would be better than "one executable".